config/data_reporter.go

```diff
@@ -0,0 +1,5 @@
+package config
+
+type HTTPDataReporterConfig struct {
```

Comment:
#PUC w/ godoc and any TODO ideas you have.

What's target URL? Prometheus? Fluent? BigQuery? Does it matter?